SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -385,6 +398,47 @@ TArray<Worker_InterestOverride> USpatialSender::CreateComponentInterest(AActor*
 	return ComponentInterest;
 }
 
+SpatialGDK::RPCsOnEntityCreation USpatialSender::PackQueuedRPCsForActor(TArray<TSharedRef<FPendingRPCParams>>* RPCList, AActor* Actor)
```

Comment:
`using namespace SpatialGDK` is at the top of this file so you can do this
```suggestion
RPCsOnEntityCreation USpatialSender::PackQueuedRPCsForActor(TArray<TSharedRef<FPendingRPCParams>>* RPCList, AActor* Actor)
```